Extension/src/LanguageServer/Providers/documentSymbolProvider.ts

```diff
@@ -16,7 +16,12 @@ export class DocumentSymbolProvider implements vscode.DocumentSymbolProvider {
         const documentSymbols: vscode.DocumentSymbol[] = [];
         if (symbols) {
             symbols.forEach((symbol) => {
-                const detail: string = util.getLocalizedString(symbol.detail);
+                let detail: string = util.getLocalizedString(symbol.detail);
+                if (symbol.scope === SymbolScope.Private) {
+                    detail = "private" + (detail.length === 0 ? "" : ": ") + detail;
```

Comment:
Do "private" and "protected" need to be localized or are they C++ keywords?

Reply:
They directly refer to C++ keywords so shouldn't be localized, but I'll look into adding a localization for the punctuation.

Reply:
I added a localizable string for the punctuation.